File: 3330. Find the Original Typed String I/sol.c
int possibleStringCount(char* word) {
    int res = 1, counter = 0, i = 1;
    char prev = word[0];
    while (word[i] != '\0') {
        if (prev == word[i])
            counter++;
        else {
            res += counter;
            counter = 0;
            prev = word[i];
        }
        i++;
    }
    res += counter;
    return res;
}

File: 118. Pascal's Triangle/sol.go
package main

func generate(numRows int) [][]int {
    res := make([][]int, numRows)
    res[0] = []int{1}
    for i:=1; i<numRows; i++ {
        res[i] = append(res[i], 1)
        prev := res[i-1]
        for j:=1; j<len(prev); j++ {
            res[i] = append(res[i], prev[j]+prev[j-1])
        }
        res[i] = append(res[i], 1)
    }
    return res
}